rllib/utils/replay_buffers/tests/test_multi_agent_prioritized_replay_buffer.py

```diff
@@ -173,8 +173,7 @@ def test_update_priorities(self):
             prioritized_replay_beta=self.beta,
             replay_mode="independent",
             replay_sequence_length=2,
-            learning_starts=0,
-            num_shards=1,
+            tnum_shards=1,
```

Comment:
typo. Why is it not caught in CI? 🤔

Reply:
**TODO**

Reply:
This is because the MultiAgentReplayBuffers accept any kwargs for forward compatibility. If we want to make them swappable, they need to accept unneeded kwargs.